Model/Payment/FastPayment.php

```diff
@@ -236,22 +236,23 @@ public function capture(InfoInterface $payment, $amount)
      */
     public function refund(InfoInterface $payment, $amount)
     {
-        $transactionId = $payment->getParentTransactionId();
+        $transactionId = $this->fastCheckoutHelper->guid() . '-' . Transaction::TYPE_REFUND;
```

Comment:
will this cause problem when by mistake if the same refund is issued twice ?

Reply:
I meant, will it prevent errors ? If somehow Fast module refund is called twice for same refund ? 
Does magento have a refundId or something ?

Reply:
Good question. Does the module get called twice for the same refund?